mitmproxy/models/http.py

```diff
@@ -22,8 +23,8 @@ def get_decoded_content(self):
         """
         ce = self.headers.get("content-encoding")
         if not self.content or ce not in encoding.ENCODINGS:
-            return self.content
-        return encoding.decode(ce, self.content)
+            return strutils.native(self.content)
```

Comment:
@mhils I couldn't decide whether `get_decoded_content` should return `bytes` or `str`, so instead of this we could keep returning bytes here, and I could make the change where the function is called!

Also, do i `strutils.native` or just plain `decode`?